static/swagger/api_v3.json

```diff
@@ -5741,6 +5741,14 @@
         "preferred": {
           "type": "boolean",
           "description": "True if the media is of high quality."
+        },
+        "image_direct_url": {
+          "type": "string",
+          "description": "Direct URL to the media's image, if there is one."
```

Comment:
How does this work with YouTube videos? Do they have `image_direct_url`s, and I presume the `view_image_url` drops to the YT page.

Reply:
I think that youtube videos should actually have this set (to the thumbnail image). See https://github.com/the-blue-alliance/the-blue-alliance-android/blob/5c94e11e5640a08a3547c16646efec3f018835c8/android/src/main/java/com/thebluealliance/androidclient/types/MediaType.java#L57

Reply:
YouTube videos don't have `image_direct_url`s and `view_image_url`s. Technically they're not images, and they have different fields for URLs: `youtube_url_link` for the link to the video and `youtube_url` for the embed URL.

Reply:
That should be okay - we can add those properties for youtube (the rendering code should do its own checks on the media type first anyway)

Reply:
Rather than adding a bunch of properties for YT (or maybe later ones) and ugly-ing the model up, can we make the fields more "generically named" and integrate them?

Reply:
So, to be clear, should YouTube media have the image properties set? I don't see any way to get YouTube thumbnails in the code. So maybe we should generalize the names of the fields so that we can comfortably support YouTube links?

Reply:
Oops. Ninja'd by @bdaroz